frontend/dockerfile/docs/experimental.md

```diff
@@ -138,3 +138,21 @@ $ buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=.
 You can also specify a path to `*.pem` file on the host directly instead of `$SSH_AUTH_SOCK`.
 However, pem files with passphrases are not supported.
 
+### RUN --security=insecure
+
+This runs the command without sandbox in insecure mode, which allows to run flows requiring elevated
+privileges (e.g. containerd). This is equivalent to running `docker run --privileged`.
+
+Default sandbox mode can be activated via `--security=sandbox`, but that is no-op.
```

Comment:
add: In order to build Dockerfiles with `--security=insecure` user needs to enable `security.insecure` entitlement when starting the buildkit daemon and allow it for a specific build request.

Reply:
updated!